### ISSUE TITLE
Mongodb 3.6 regression when upgrading from 0.9.1 to 0.9.2

### ISSUE THREAD

**Issue by 1st8** [zookzook/elixir-mongodb-driver#155]
Hi,

i have a green test suite when running mongodb_driver 0.9.1 against mongodb 3.6.

After upgrading to driver 0.9.2 the tests hang and time out with the following message:

```
     ** (exit) exited in: GenServer.call(:mongo, {:checkout_session, :write, [write_counter: 1]}, 60000)
         ** (EXIT) time out
     stacktrace:
       (elixir 1.10.4) lib/gen_server.ex:1023: GenServer.call/3
       (mongodb_driver 0.9.2) lib/mongo/session.ex:142: Mongo.Session.start_session/3
       (mongodb_driver 0.9.2) lib/mongo/session.ex:694: Mongo.Session.in_session/5
       (mongodb_driver 0.9.2) lib/mongo.ex:1482: Mongo.issue_command/4
       (mongodb_driver 0.9.2) lib/mongo.ex:849: Mongo.insert_one/4
```

Might be related to #103 

When upgrading MongoDB to 4.4 everything works again.

TL;DR:

|  | 0.9.1 | 0.9.2 | 
|---|---|---|
| MongoDB 3.6 | ✅ | 🔴 | 
| MongoDB 4.4 | ✅ | ✅ | 

Let me know if I can assist you with this.

Cheers!

**Comment by zookzook** [comment]
Can you test the master branch? I think, the next release will fix it (https://github.com/zookzook/elixir-mongodb-driver/pull/147)

**Comment by 1st8** [comment]
Thank you for the quick response. You're right, it works with the current master.

**Comment by zookzook** [comment]
Fixed in the next coming release 1.0.0
